.github/VelogCloneCoding.css

```
@@ -0,0 +1,110 @@
+
+.h2{
+    font-family: monospace;
+    font-weight : 700;
+}
+
+#header{
+    width : 100%;
+    height: 80px;
+    margin: 10px 135px;
+    
+}
+
+/* hover 상태에서 테두리 색상 변경 */
+
+button:hover{
+    color: skyblue;
+    border-color: skyblue; 
+}
+  
+
+/* velog 로고, 검색, 새 글 작성, 프로필 container  */
+
+.firstrow-container {
+    display: inline-flex; 
+    align-items: center;
+ 
+  }
+
+.button-searching {
+    margin-top: 5px;
+    margin-left: 1300px; 
+    background-color: white;
+    border:none;
+
+}
+
+.button-new-article{
+
+    border-radius: 20px;
+    width: 80px;
+    height: 30px;
+    font-weight: bold;
+    background-color: white;
+
+}
+
+.button-my-account{
+    background-color: white;
+    border:none;
+
+}
+  
+
+
+/* 트렌딩, 최신, 검색 결과 시기 container */
+
+.button-container {
+    display: inline-flex;
+    align-items:flex-end;
+  }
+  
+.button-trending {
+    border: none; /* 다른 테두리 비활성화 */
```

여기에 
```suggestion
  display: flex;
  align-items: center;
```

를 추가해주면, frog 게임에서 했던 것처럼 button-trending 태그 안에 있는 텍스트와 icon이 서로 가운데를 중심으로 정렬되어 일자로 배치되는 것을 확인하실 수 있습니다.